src/clients/angular/buildy-app/src/app/components/build-form/build.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="container">
+  <mat-form-field>
+    <mat-label>Filter</mat-label>
+    <input matInput (keyup)="applyFilter($event)" placeholder="Ex. Processor" #input>
+  </mat-form-field>
+
+  <div class="mat-elevation-z8">
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <!-- Part Column -->
+      <ng-container matColumnDef="part">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Part </th>
+        <td mat-cell *matCellDef="let row"> {{ row.part }} </td>
+        <td mat-footer-cell *matFooterCellDef> Total </td>
+      </ng-container>
+
+      <!-- Selected part Column -->
+      <ng-container matColumnDef="selectedPart">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Selected part </th>
+        <td mat-cell *matCellDef="let row"> {{ row.selectedPart }}</td>
+        <td mat-footer-cell *matFooterCellDef></td>
+      </ng-container>
+
+      <!-- Price Column -->
+      <ng-container matColumnDef="price">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Price </th>
+        <td mat-cell *matCellDef="let row"> {{ row.price | currency }} </td>
+        <td mat-footer-cell *matFooterCellDef>{{ getTotalPrice() | currency }}</td>
```

Comment:
Rewrite this and avoid using method call from html. It gets called multiple times.